### DISCUSSION TITLE
WikiJS update script

### DISCUSSION THREAD

**Discussion by ChrisGoudt** [tteck/Proxmox#912] [Help Needed] [answered]
When I try to update the WikiJS LXC, I get this error:

 `✓ Stopped Wikijs
 - Updating Wikijs...cp: cannot stat '/opt/wiki/config.yml': No such file or directory
‼ ERROR 1@321 Unknown failure occurred.`

This seems to be caused by an incorrect path, in my installation, the correct path is /opt/wiki**js**/config.yml

**Comment by tteck** [comment, marked as answer]
Thanks for reporting! FIXED

**Reply by ChrisGoudt** [reply]
That was quick :) Thanks
